packages/manager/modules/emailpro/src/account/emailpro-account.controller.js

```diff
@@ -254,6 +255,7 @@ export default /* @ngInject */ (
       primaryEmailAddress: newConfiguredAccount,
     })
       .then((account) => {
+        $scope.accountIds = $scope.accountIds.slice(1);
```

Comment:
are we sure we always want to omit the first element ? shouldn't we check depending on a `domainName` or something else ?

Reply:
@radireddy Do you have any information to provide regarding marie's comment?
Thank you!

Reply:
ping @radireddy

Reply:
It is ok to drop the first one after it is used. Whenever mxplan email is created, the id is picked from the pool. Once the id is picked and used, it should not be picked again. which was the original issue.